meta-oe/recipes-devtools/cereal/cereal_1.0.bb

```diff
@@ -24,3 +26,11 @@ EXTRA_OECMAKE += "\
 "
 
 S = "${WORKDIR}/git"
+
+PROVIDES += "${PN}-dev"
```

Comment:
cereal provides only header files for development, no binary or library is created during build. hence reflect that by providing dev package